# Message describing the command joint angles
Header header
string[] names
float32[] angles